provider_sources/_compress_providers.py

```diff
@@ -84,131 +84,116 @@ def update_year(provider_or_tile):
-            name = "parcels"
-
-        if "Style" in layer:
-            if type(layer["Style"]) is dict:
-                style = layer["Style"]["ows:Identifier"]
-
-            elif type(layer["Style"]) is list:
-                style = layer["Style"][1]["ows:Identifier"]
-        else:
-            style = "normal"
-
-        TileMatrixSetLimits = layer["TileMatrixSetLink"]["TileMatrixSetLimits"][
-            "TileMatrixLimits"
-        ]
-        min_zoom = int(TileMatrixSetLimits[0]["TileMatrix"])
-        max_zoom = int(TileMatrixSetLimits[-1]["TileMatrix"])
-        TileMatrixSet = layer["TileMatrixSetLink"]["TileMatrixSet"]
-        format = layer["Format"]
-        bounding_lowerleft_corner = layer["ows:WGS84BoundingBox"][
-            "ows:LowerCorner"
-        ]  # given with lon/lat order
-        bounding_upperright_corner = layer["ows:WGS84BoundingBox"][
-            "ows:UpperCorner"
-        ]  # given with lon/lat order
-        lowerleft_corner_lon, lowerleft_corner_lat = bounding_lowerleft_corner.split(
-            " "
-        )
-        upperright_corner_lon, upperright_corner_lat = bounding_upperright_corner.split(
-            " "
-        )
-        bounds = [
-            [float(lowerleft_corner_lat), float(lowerleft_corner_lon)],
-            [float(upperright_corner_lat), float(upperright_corner_lon)],
-        ]
-
-        if format == "application/x-protobuf":
-            pass
-        elif format == "image/x-bil;bits=32":
-            pass
-        elif apikey == "lambert93":
-            pass
-        else:
-            tilelayers_list.append("GeoportailFrance." + name)
-            leaflet["GeoportailFrance"][name] = {
-                "url": """https://wxs.ign.fr/{apikey}/geoportail/wmts?REQUEST=GetTile&SERVICE=WMTS&VERSION=1.0.0&STYLE={style}&TILEMATRIXSET={TileMatrixSet}&FORMAT={format}&LAYER={variant}&TILEMATRIX={z}&TILEROW={y}&TILECOL={x}""",
-                "html_attribution": """<a target="_blank"href="https://www.geoportail.gouv.fr/">Geoportail France</a>""",
-                "attribution": "Geoportail France",
-                "bounds": bounds,
-                "min_zoom": min_zoom,
-                "max_zoom": max_zoom,
-                "apikey": apikey,
-                "format": format,
-                "style": style,
-                "variant": variant,
-                "name": "GeoportailFrance." + name,
-                "TileMatrixSet": TileMatrixSet,
-            }
-
-            possibly_broken = [
-                "Ocsge_Constructions_2002",
-                "Ocsge_Constructions_2014",
-                "Ocsge_Couverture_2002",
-                "Ocsge_Couverture_2014",
-                "Ocsge_Usage_2002",
-                "Ocsge_Usage_2014",
-                "Orthoimagery_Orthophotos_Coast2000",
-                "Pcrs_Lamb93",
-                "Orthoimagery_Ortho_sat_Spot_2013",
-                "Orthoimagery_Orthophotos_1980_1995",
-            ]
-
```

Comment:
possibly_broken = [
                "Ocsge_Constructions_2002",
                "Ocsge_Constructions_2014",
                "Pcrs_Lamb93",
            ]

Reply:
Maybe a global and systematic check of broken `GeoportailFrance` broken is required